cmd/tsbs_load_mongo/main.go

```diff
@@ -76,23 +77,29 @@ func init() {
 	retryableWrites = viper.GetBool("retryable-writes")
 	orderedInserts = viper.GetBool("ordered-inserts")
 	randomFieldOrder = viper.GetBool("random-field-order")
+	batchMetaFields = viper.GetBool("batch-meta-fields")
 	collectionSharded = viper.GetBool("collection-sharded")
 	numInitChunks = viper.GetUint("number-initial-chunks")
 	shardKeySpec = viper.GetString("shard-key-spec")
 	balancerOn = viper.GetBool("balancer-on")
 	metaFieldIndex = viper.GetString("meta-field-index")
 	granularity = viper.GetString("granularity")
 	
-	if documentPer {
-		config.HashWorkers = false
-	} else {
-		config.HashWorkers = true
+	config.HashWorkers = true
```

Comment:
Would we want to make it so that this is only true when `documentPer` or if `batchMetaFields` is `true`?
I think hashWorkers should be set to true by default so that we have numChannels match the number of workers rather than there to be only one channel with channel capacity * number of workers (the case when `hashWorkers` is `false`). The code related to this behavior is [linked here](https://github.com/mongodb-forks/tsbs/blob/main/load/loader.go#L186) and [here](https://github.com/mongodb-forks/tsbs/blob/main/load/loader.go#L110)